core/vm/contract.go

```diff
@@ -49,24 +49,21 @@ type Contract struct {
 	caller        ContractRef
 	self          ContractRef
 
-	jumpdests destinations // result of JUMPDEST analysis.
+	jumpdests destinations // Aggregated result of JUMPDEST analysis.
```

Comment:
Could we drop the `destinations` typdef altogether? Previously we at least had a methof on it, but now it's just a `map[common.Hash]bitvec`. There's not much point really to keep a type just for this.